.github/workflows/build_conda_pkg.yml

```diff
@@ -0,0 +1,66 @@
+name: Build Conda Package
+
+on:
+  pull_request:
```

Comment:
I think we only want this to run on pushes to main

Reply:
@freddyaboulton Currently `build_conda_pkg` runs in CircleCI as part of the other workflows, so on every commit, primarily for the `shellcheck`. Are we changing this behaviour @dsherry ?

Reply:
Ah, I think I missed `if: github.ref == 'refs/head/main'` line below. If that means that the package is only built on commits to main, then I think we're good to go!

But I think we can make it simpler by using `on: push: branches: -main`? I don't think we need the shellcheck to run on PR branches but let me know if I'm misunderstanding!

Reply:
Got it, I was under the impression we were keeping the same behaviour as the CircleCI implementation but if we're good to not do a shellcheck on PR branches then I can definitely make the change to only run on `main`. I agree that I don't think it's necessary.

Reply:
Sounds good to me!

I also wonder if shellcheck would belong better in our `lint` jobs.